qemu/deps/performance/start_testpmd.py

```diff
@@ -23,18 +25,21 @@ class TestPMD(object):
     def __init__(self):
 
         self.proc = None
-        testpmd_cmd = "/usr/bin/testpmd "
+        testpmd_cmd = subprocess.check_output(
```

Comment:
why not `process.getoutput()`?

Reply:
this script runs on guest, there is avocado frame.

Reply:
> this script runs on guest, there is avocado frame.

I see, thanks for the clarification.